src/Microsoft.SqlTools.ServiceLayer/Profiler/ProfilerSessionMonitor.cs

```diff
@@ -72,9 +65,6 @@ public void AddSessionListener(IProfilerSessionListener listener)
             }
         }
 
-        /// <summary>
-        /// Start monitoring the provided session
-        /// </summary>
         public bool StartMonitoringSession(string viewerId, IXEventSession session)
```

Comment:
It seems this method will always return true, you can change its return type to be void.